test/MOIWrapper.jl

```diff
@@ -57,4 +57,24 @@ end
             MOIT.TestConfig()
         )
     end
+end
+
+@testset "ModelLike tests" begin
+    solver = CPLEX.Optimizer()
+    # TODO add a test config to disable testing problem name.
+    # @testset "nametest" begin
+    #     MOIT.nametest(solver)
```

Comment:
Why does the nametest fail? Since CLPEX uses LQOI it should pass...

Reply:
You're right. My bad.